lib/test.gi

```diff
@@ -931,9 +931,9 @@ InstallGlobalFunction( "TestDirectory", function(arg)
 
   if opts.exitGAP then
     if testTotalFailures = 0 then
-      QUIT_GAP(0);
+      QuitGap(0);
     else
-      QUIT_GAP(1);
+      QuitGap(1);
```

Comment:
Independent of this PR: looking at this PR, I discovered `SetExitValue` and learned that one could rewrite the above code as `QuitGap(testTotalFailures = 0);`. But I am not aware of *any* code using that.  So, is it worth keeping this complication? If so, shouldn't we then actually use it here and elsewhere (perhaps not as part of this PR)?
On the other hand, it seems to me that code like `QuitGap(testTotalFailures = 0);`

Reply:
So, I have used SetExitValue(1), so if GAP exits in the middle of my program (say because of an error), the exit value ends up set to a non-zero value.

Using QUIT_GAP(boolean), I don't remember ever seeing. We could probably remove it.

Reply:
Looking at packages, there are a couple of `QUIT_GAP()`, but no passing in of booleans I can see.

Reply:
I agree that `SetExitValue` is useful to have, yes.